understanding/no-keyboard-trap.html

```diff
@@ -58,126 +56,103 @@
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 2.1.2:</span>No Keyboard Trap (Level A)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 2.1.2:</span>キーボードトラップなし (レベル A)
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>If keyboard focus can be moved to a component of the page using a <a href="#dfn-keyboard-interface">keyboard interface</a>, then focus can be moved away from that component using only a keyboard interface,
-                     and, if it requires more than unmodified arrow or tab keys or other standard exit
-                     methods, the user is advised of the method for moving focus away.
+                  <p><a href="#dfn-keyboard-interface">キーボードインタフェース</a>を用いてキーボードフォーカスをそのウェブページのあるコンポーネントに移動できる場合、キーボードインタフェースだけを用いてそのコンポーネントからフォーカスを外すことが可能である。さらに、修飾キーを伴わない矢印キー、Tab キー、又はフォーカスを外すその他の標準的な方法でフォーカスを外せない場合は、フォーカスを外す方法が利用者に通知される。
                      
                   </p>
                   <div class="note">
-                     <p class="note-title marker">Note</p>
-                     <p>Since any content that does not meet this success criterion can interfere with a user's
-                        ability to use the whole page, all content on the Web page (whether it is used to
-                        meet other success criteria or not) must meet this success criterion. See <a href="#cc5">Conformance Requirement 5: Non-Interference</a>.
+                     <p class="note-title marker">注記</p>
+                     <p>この達成基準を満たさないコンテンツでは、利用者がそのウェブページ全体を使用できない恐れがあるため、ウェブページ上のすべてのコンテンツは他の達成基準を満たすために用いられているか否かにかかわらず、この達成基準を満たさなければならない。<a href="https://waic.jp/translations/WCAG22/#cc5">適合要件 5: 非干渉</a>を参照。
                         
                      </p>
                   </div>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Keyboard users don't get stuck.</dd>
+                  <dt>目標</dt>
+                  <dd>キーボード利用者が、はまって動けなくならないようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Ensure users always know how to navigate away from components.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>コンポーネントから離れる方法を、利用者が常に知っているようにする。.</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>People who rely on the keyboard often have no other means to navigate.</dd> 
+                  <dt>なぜそれが重要か</dt>
+                  <dd>キーボード操作に依存する人は、ナビゲートする別の手段を持ち合わせていないことが多い。</dd> 
```

Comment:
Goo辞書によると
https://dictionary.goo.ne.jp/word/%E6%8C%81%E5%90%88%E3%81%9B%E3%82%8B/#jn-219266
> もち‐あわ・せる〔‐あはせる〕【持（ち）合（わ）せる】
> ちょうどそのときに持っている。

となっており、タイミングによっては別の手段があるようにも読めてしまうのでは？
ということで、シンプルに下記はどうでしょう？

```suggestion
                  <dd>キーボード操作に依存する人は、ナビゲートする別の手段がないことが多い。</dd> 
```

Reply:
「持ち合わせる」のニュアンス、なるほどです。シンプルに「別の手段がないことが多い」で十分かと思います！